api/app.js

```diff
@@ -141,18 +146,16 @@ async function report() {
         "rec2XI8QAsPr7EMVB"
       ]
 	})
-	let newStats = await getStats();
-	averageSpeed = newStats.fields.average.toFixed(3);
+	let averageSpeed = await getAverage();
 	let thousandsGoal = Math.ceil(latest / 1000) * 1000;
-	let thousandsTime = predictTime(thousandsGoal, latest);
+	let thousandsTime = predictTime(thousandsGoal, latest, averageSpeed);
 	let tenThousandsGoal = Math.ceil(latest / 5000) * 5000;
 	let pastThousandsGoal = Math.floor(latest / 1000) * 1000;
-	let tenThousandsTime = predictTime(tenThousandsGoal, latest);
+	let tenThousandsTime = predictTime(tenThousandsGoal, latest, averageSpeed);
 	let message =
 		`Good evening, my underlings! You have done a _brilliant_ job with the counting. Mmm. Yes. Numbers are savory. 
 		Today we've went from *${oldest}* to *${latest}*!
-		- :arrow_upper_right: The day's progress: *+${diff}*
```

Comment:
Um... I don't think we should delete that? It shows our progress that day

Reply:
yeah good idea. I was having wayyy to many issues with git for some reason. It kept deleting lines. I'll try re-adding that line through GitHub web.

Reply:
Good catch!